app/models/ingredient.rb

```diff
@@ -1,3 +1,9 @@
 class Ingredient < ApplicationRecord
+   belongs_to :recipe
+   has_many :recipe_ingredients
+   has_many :recipes, through: :recipe_ingredients
+
```

Comment:
Remember that when adding a `belongs_to` relationship, it's because there is a foreign key on that table. I do not recall there being a foreign key for a recipe on the ingredients table. You wouldn't often see a `has_many` and a `belongs_to` pointing to the same or similar resources, in this example `:recipe` and `:recipes`.

The tests for the association were already created, all you needed to do was add those associations in their respective models to get all the tests to pass.